Kudu.Services/Diagnostics/LogStreamManager.cs

```diff
@@ -74,10 +75,14 @@ public IAsyncResult BeginProcessRequest(HttpContext context, AsyncCallback cb, o
             });
 
             string path = ParseRequest(context);
-            if (!Directory.Exists(path))
+            if (!Directory.Exists(path) && !context.Request.IsFunctionPortalRequest())
```

Comment:
I figured it's technically a breaking change. Not sure why someone would've relied on the old behavior, but if you think it's okay to always create the folder if it doesn't exist, then I can change it to do that.